src/app.py

```diff
@@ -4337,6 +4338,10 @@ def entity_bulk_update():
 
     validate_token_if_auth_header_exists(request)
     require_json(request)
+    try:
+        schema_validators.validate_application_header_before_property_update(request)
```

Comment:
@ChuckKollar we do not need to explicitly call the validator here. When you make the PUT calls against entity-api itself to update each dataset/upload, it'll go through the yaml validation: https://github.com/hubmapconsortium/entity-api/blob/8b3e20f9e25a8542bacfd768abbdb71f0fcb86c6/src/app.py#L1165

Are you not getting the 400 response without this explicit handling?

Reply:
When I remove the EXPLCIT CALL I am NOT getting an error when
 --header "X-Hubmap-Application: entity-apix"
 for example.
So, it would appear that the explicit call to the validator needs to be there.